### ISSUE TITLE
Extra form field - Food details page - URL field is not display

### ISSUE THREAD

**Issue by mistry-jignesh** [wpfoodmanager/wp-food-manager#123]
Extra form field - Food details page - URL field is not display.

![image](https://user-images.githubusercontent.com/75515088/196657611-de9cc5f9-e1aa-4b68-b58c-8aad85e4e450.png)

**Comment by kushang-tailor** [comment]
Hello @mistry-jignesh ,

I have added the New section for the **Extra Form fields** called **Additional Details for Extra Toppings**. So you can check and verify those fields under this section.

**Comment by mistry-jignesh** [comment]
Verified this issue . Additional Details for Extra Toppings section URL filed is not display.

Below URL I have added in the  Extra Toppings section.

![image](https://user-images.githubusercontent.com/75515088/200471962-42876c7e-f74c-4f20-a739-1f49424137ba.png)

URL filed is not display on the details page.

![image](https://user-images.githubusercontent.com/75515088/200472017-dbf8706c-ac94-4568-8c30-301345596bff.png)
